metrics/web/lib/common/presentation/widgets/metrics_text_form_field.dart

```diff
@@ -0,0 +1,60 @@
+import 'package:flutter/material.dart';
+
+/// Displays a metrics text form field widget.
+class MetricsTextFormField extends StatelessWidget {
+  final String label;
+  final TextEditingController controller;
+  final String Function(String) validator;
+  final bool isClearable;
+
+  /// Creates a widget that represents a specific version of a [TextFormField].
+  ///
+  /// The text field has the given [label]. 
+  /// The text field is controlled by the [controller]. 
+  /// The [validator] callback allows to validate the entered text. 
+  /// Setting the [isClearable] argument to [true] allows clearing a text inside the input.
+  /// The [isClearable] argument is [false] by default.
+  /// 
+  /// The [label] and the [controller] arguments should not be null.
+  const MetricsTextFormField({
+    @required this.label,
+    @required this.controller,
+    this.validator,
+    this.isClearable = false,
+  })  : assert(label != null),
+        assert(controller != null);
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: <Widget>[
+        Padding(
+          padding: const EdgeInsets.only(bottom: 8.0),
+          child: Text(label),
+        ),
```

Comment:
Vlad, we've implemented it by the old design template, and it didn't look like the InputDecoration.label. For now, in the new design, it looks like the label text is missing, and it's just the InputDecoration.hint text, so I have a question. Should we fix it now or when we are going to implement the new design?

Reply:
I think it would be better to fix it now using an `InputDecoration.label`. Then we can change it to hint when we'll implementing the new design. 

cc @illia-romanenko - what do you think?